### ISSUE TITLE
./tools/run_tests/run_tests.py fails with "php does not support multiple build configurations"

### ISSUE THREAD

**Issue by jcanizales** [grpc/grpc#1040]
We should fix it or amend the documentation in CONTRIBUTING.md.

**Comment by jcanizales** [comment]
Note, this is running on a Mac from master.

**Comment by murgatroid99** [comment]
So, this is caused by a change I made a week or two ago to run_tests.py. Basically, I modified it so that we could run it with different configurations and scripting languages (`-l node -c dbg`, for example). But because all of the build steps run before all of the tests, running it with multiple configurations would only actually run the tests with the last one. Rather than make the test output misleading, I made it throw an error.

So, basically, we should probably do some more work on run_tests.py, and we should also probably change CONTRIBUTING,md.

**Comment by ctiller** [comment]
How about we just change the default configs to just opt?

**Comment by nicolasnoble** [comment]
I agree with @ctiller. It makes sense to change the default configs to just opt here.

**Comment by nicolasnoble** [comment]
I have changed the run_tests.py script to only run opt by default. Could you confirm this is fixed and close the issue if it is ?

**Comment by murgatroid99** [comment]
I can run `run_tests.py` without this sort of error, so I'd say it's fixed.
